mix.exs

```diff
@@ -101,7 +101,10 @@ defmodule Livebook.MixProject do
       {:protobuf, "~> 0.8.0"},
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:floki, ">= 0.27.0", only: :test},
-      {:bypass, "~> 2.1", only: :test}
+      {:bypass, "~> 2.1", only: :test},
+      # ZTA deps
+      {:jose, "~> 1.11.5"},
+      {:req, "~> 0.3.8"}
```

Comment:
Is there a reason we can't use `Livebook.Utils.HTTP`?

Reply:
It doesn't parse the way we need and we will remove ZTA from Livebook soon